lambdaclass/zksync_seaport
# Store and retrieve executionhelper deployment address

Closes #10 

## Steps for manual testing

1. Clone the repo (to get a clean state) and checkout this branch.
2. Run `cd ExecutionHelper`
3. Run `yarn install`
4. Run `yarn hardhat compile`
5. Run `yarn hardhat deploy-zksync --script deploy.ts`
6. Run `cat .executionHelper.address`. **You should see the same address as the one printed by the deployment command in the previous step**
7. Run `cd ..` to get back into the repo's root dir.
8. Run `yarn install`
9. Run `yarn hardhat compile` **This step will fail immediately in case `.executionHelper.address` does not exist**
